Numbers/ViewController/GridViewController.swift

```diff
@@ -0,0 +1,296 @@
+//
+//  GridViewController.swift
+//  Numbers
+//
+//  Created by Brooklyn Gibbs on 6/19/19.
+//  Copyright © 2019 Brooklyn Gibbs. All rights reserved.
+//
+
+import UIKit
+import CoreData
+
+// two in a row
+class GridViewController: UIViewController {
+    
+    @IBOutlet weak var box1: UILabel!
+    @IBOutlet weak var box2: UILabel!
+    @IBOutlet weak var box3: UILabel!
+    @IBOutlet weak var box4: UILabel!
+    @IBOutlet weak var box5: UILabel!
+    @IBOutlet weak var box6: UILabel!
+    @IBOutlet weak var box7: UILabel!
+    @IBOutlet weak var box8: UILabel!
+    @IBOutlet weak var box9: UILabel!
+    @IBOutlet weak var box10: UILabel!
+    @IBOutlet weak var box11: UILabel!
+    @IBOutlet weak var box12: UILabel!
+    
+    @IBOutlet weak var incorrectLabel: UILabel!
+    @IBOutlet var tapGesture1: UITapGestureRecognizer!
+    
+    var boxList = [UILabel]()
+    var boxOrder = [Int]()
+    var level = 4
+    @IBOutlet weak var levelLabel: UILabel!
+    var tapCount = 0
+    var livesLeft = 2
+    let secs = 0.7
+    
+    // Screen width.
+    public var screenWidth: CGFloat {
+        return UIScreen.main.bounds.width
+    }
+    
+    // Screen height.
+    public var screenHeight: CGFloat {
+        return UIScreen.main.bounds.height
+    }
+    
+    
+    override func viewDidLoad() {
+        incorrectLabel.isHidden = true
+        super.viewDidLoad()
+        setGrid(color: UIColor.black.cgColor)
+        setArray()
+        setLevel()
+        tapGesture1.isEnabled = false
```

Comment:
This should probably move to the top of the function, right under super.viewDidLoad